resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -325,6 +325,12 @@ private[spark] object Config extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val KUBERNETES_SUBMIT_GRACE_PERIOD =
+    ConfigBuilder("spark.kubernetes.submitGracePeriod")
+      .doc("Number of seconds that should be used to wait for graceful deletion of Spark pods.")
+      .longConf
```

Comment:
This should be a `timeConf`, and the doc should not specify a unit.

Reply:
Ok will fix just but grace period should be small actually it does not make much sense to have an arbitrary time unit like days or something. @vanzin that is why I set seconds "The kubectl delete command supports the --grace-period=<seconds>" https://kubernetes.io/docs/concepts/workloads/pods/pod/

Reply:
Ok I will set unit in `timeConf` to sec like [`kubectl`](https://kubernetes.io/docs/concepts/workloads/pods/pod) does  `kubectl delete command supports the --grace-period=<seconds>`. In general passing arbitrary time strings does not make much sense as grace period is small eg. not days.

Reply:
`timeConf` makes it consistent with the rest of Spark. And avoids having to tell the user what is the unit, since the user gets to say it ("30s", "2min").

Reply:
Yes I know the benefits although for that  property a more constraint value is more aligned with the k8s use.